File: be/src/main/java/com/project/popupmarket/dto/user/UserDto.java
package com.project.popupmarket.dto.user;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@JsonInclude(JsonInclude.Include.NON_NULL)
public class UserDto {
    private Long id;
    private String email;
    private String name;
    private String brand;
    private String profileImage;
    private String tel;
}

File: be/src/main/java/com/project/popupmarket/dto/user/UserDto.kt
package com.project.popupmarket.dto.user

import com.project.popupmarket.enums.Role

data class UserDto(
    var id: Long? = null,
    var email: String? = null,
    var name: String? = null,
    var brand: String? = null,
    var role : Role? = null,
    var profileImage: String? = null,
    var tel: String? = null,
    var businessId: String? = null
)